pinot-query-planner/src/main/java/org/apache/pinot/query/planner/logical/RexExpressionUtils.java

```diff
@@ -248,9 +248,11 @@ private static List<RexExpression> toFunctionOperands(RexInputRef rexInputRef, S
   public static RexExpression fromAggregateCall(AggregateCall aggregateCall) {
     List<RexExpression> operands =
         aggregateCall.getArgList().stream().map(RexExpression.InputRef::new).collect(Collectors.toList());
+    List<RexExpression> preArg =
+        aggregateCall.rexList.stream().map(RexExpressionUtils::fromRexNode).collect(Collectors.toList());
```

Comment:
Since it won't be backward compatible anyway, can we directly use this as `operands`? I think that is the general way to handle it, where we already support reading literal from operands

Reply:
This won't work for intermediate stage, which expects only one argument in the operand list with InputRef to the type of an intermediate object.